CalculatorIdo/Script.js

```diff
@@ -0,0 +1,149 @@
+window.onload = () => {clearCalculator();}
+
+var currAns = '';
```

Comment:
Use only let and const, var became sort of deprecated in the recent versions of javascript